content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -65,57 +67,11 @@ populate with that information.
 
 Under the :guilabel:`Customer` heading, choose from the following options:
 
-- :guilabel:`Create a new customer`: Choose this option to use the information in the lead to create
+- :guilabel:`Create a new customer`: choose this option to use the information in the lead to create
   a new customer.
-- :guilabel:`Link to an existing customer`: Choose this option, then select a customer from the
+- :guilabel:`Link to an existing customer`: choose this option, then select a customer from the
   resulting drop-down menu, to link this opportunity to the existing customer record.
-- :guilabel:`Do not link to a customer`: Choose this option to convert the lead, but not link it to
+- :guilabel:`Do not link to a customer`: choose this option to convert the lead, but not link it to
   a new or existing customer.
 
 Lastly, when all configurations are complete, click :guilabel:`Create Opportunity`.
```

Comment:
Do you want to mention where this newly-created opportunity can then be found within the application? That could help round out this succinct doc. What do you think?